src/js/actions/search/commands.js

```diff
@@ -123,13 +123,11 @@ define(function (require, exports) {
     var _menuCommandSearchOptions = function () {
         var menuStore = this.flux.store("menu"),
             menu = menuStore.getApplicationMenu(),
-            menuMap = menu.rootMap,
             roots = menu.roots.reverse();
 
         var menuCommands = [];
         roots.forEach(function (root) {
-            var currRoot = menuMap.get(root.id),
```

Comment:
@iwehrman @mcilroyc I agree about it being a misunderstanding -- it is odd that it was working before and I had the same question of why this was here in the first place. But yes both of your observations are correct, it was just a small mistake in what reference was being inserted into the array.